docs/docs/consumer.mdx

```diff
@@ -70,32 +72,27 @@ To specify a CA bundle pass the ca parameter:
 from zepben.evolve import connect_tls, SyncNetworkConsumerClient, Feeder
 from zepben.protobuf.nc.nc_requests_pb2 import INCLUDE_ENERGIZED_LV_FEEDERS
 
-with open('path/to/ca/bundle', 'rb') as f:
-    ca = f.read()
-
-    with connect_tls(host="ewb.zepben.com", rpc_port=443, ca=ca) as channel:
-        client = SyncNetworkConsumerClient(channel)
+async with connect_tls(host="ewb.zepben.com", rpc_port=443, ca_filename="path/to/ca/bundle") as channel:
```

Comment:
See my comment on the other PR, but we shouldn't be using with or async with when using a SyncNetworkConsumerClient.
If removing the with works, do that, and I guess just manually close the channel at the end.

Reply:
The `close` method of a `grpc.aio.Channel` is also async.

Reply:
Updated to `channel = ...`. Also fixed ordering of some parameters (technically would work because they are kwargs, but canonical is best)